### ISSUE TITLE
error determining github context

### ISSUE THREAD

**Issue by ofrobots** [googleapis/releasetool#52]
WIth v0.7.0:

```
~/src/veneer/nodejs-logging master
❯ releasetool start
o/ Hey, ofrobots, let's release some stuff!
> Determining GitHub context.
Traceback (most recent call last):
  File "/Users/ofrobots/homebrew/bin/releasetool", line 11, in <module>
    sys.exit(main())
  File "/Users/ofrobots/homebrew/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/Users/ofrobots/homebrew/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Users/ofrobots/homebrew/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/ofrobots/homebrew/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/ofrobots/homebrew/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Users/ofrobots/homebrew/lib/python3.6/site-packages/releasetool/main.py", line 101, in start
    return releasetool.commands.start.nodejs.start()
  File "/Users/ofrobots/homebrew/lib/python3.6/site-packages/releasetool/commands/start/nodejs.py", line 229, in start
    releasetool.commands.common.setup_github_context(ctx)
  File "/Users/ofrobots/homebrew/lib/python3.6/site-packages/releasetool/commands/common.py", line 84, in setup_github_context
    _determine_origin(ctx)
  File "/Users/ofrobots/homebrew/lib/python3.6/site-packages/releasetool/commands/common.py", line 37, in _determine_origin
    origin = remotes["origin"]
KeyError: 'origin'

~/src/veneer/nodejs-logging master
❯ git remote -v
ofrobots        git@github.com:ofrobots/nodejs-logging.git (fetch)
ofrobots        git@github.com:ofrobots/nodejs-logging.git (push)
origin  https://github.com/googleapis/nodejs-logging (fetch)
origin  https://github.com/googleapis/nodejs-logging (push)
```

LMK if any more info is needed.

**Comment by busunkim96** [comment]
@ofrobots Could you also do a `git config --list`?

**Comment by JustinBeckwith** [comment]
Pulling in @tseaver who was also having similar issues

**Comment by ofrobots** [comment]
@busunkim96 here you go

```
credential.helper=osxkeychain
user.name=Ali Ijaz Sheikh
user.email=ofrobots@google.com
user.signingkey=5BD191B0Dredacted40B073C601
alias.lol=log --graph --decorate --pretty=oneline --abbrev-commit
alias.lola=log --graph --decorate --pretty=oneline --abbrev-commit --all
color.branch=auto
color.diff=auto
color.interactive=auto
color.status=auto
core.editor=code --wait
core.excludesfile=/Users/ofrobots/.gitignore_global
http.cookiefile=/Users/ofrobots/.gitcookies
gpg.program=/Users/ofrobots/homebrew/bin/gpg
```

**Comment by busunkim96** [comment]
@ofrobots Sorry, I should have been more specific.  Could you do `git config --list` from within your `nodejs-logging` directory? (The github context is constructed from the results of that command.)

**Comment by ofrobots** [comment]
```
credential.helper=osxkeychain
user.name=Ali Ijaz Sheikh
user.email=ofrobots@google.com
user.signingkey=5Bredacted601
alias.lol=log --graph --decorate --pretty=oneline --abbrev-commit
alias.lola=log --graph --decorate --pretty=oneline --abbrev-commit --all
color.branch=auto
color.diff=auto
color.interactive=auto
color.status=auto
core.editor=code --wait
core.excludesfile=/Users/ofrobots/.gitignore_global
http.cookiefile=/Users/ofrobots/.gitcookies
gpg.program=/Users/ofrobots/homebrew/bin/gpg
core.repositoryformatversion=0
core.filemode=true
core.bare=false
core.logallrefupdates=true
core.ignorecase=true
core.precomposeunicode=true
remote.origin.url=https://github.com/googleapis/nodejs-logging
remote.origin.fetch=+refs/heads/*:refs/remotes/origin/*
branch.master.remote=origin
branch.master.merge=refs/heads/master
remote.ofrobots.url=git@github.com:ofrobots/nodejs-logging.git
remote.ofrobots.fetch=+refs/heads/*:refs/remotes/ofrobots/*
```

**Comment by busunkim96** [comment]
@ofrobots Apologies, I think I made a mistake when I released v0.7.0. 🙄Please try with v0.8.0. Thank you!
